roles/openshift_metrics/defaults/main.yaml

```diff
@@ -4,8 +4,14 @@ image_version: latest
 master_url: https://kubernetes.default.svc.cluster.local
 
 hawkular_user_write_access: False   #if user accounts should be able to write metrics
+hawkular_cassandra_nodes: 1
+# emptydir|pv|dynamic
+hawkular_cassandra_storage_type: emptydir
+hawkular_cassandra_pv_prefix: metrics-cassandra
+hawkular_cassandra_pv_size: 10Gi
 
 heapster_standalone: False
+heapster_allowed_users: system:master-proxy
```

Comment:
Agreed.  The README was still the original one, so I went ahead and updated it
now that we have a semi-stable version.  I'm not sure about some of the things
there (license?  authors?), so I didn't touch them.

We have a lot of documentation on these parameters (and other deployer-related
topics) in [origin-metrics][origin_metrics], and logging is probably similar.
Are we keeping them there?  Or maybe documenting them here or in
openshift-docs?

[origin_metrics]: https://github.com/openshift/origin-metrics